how-we-work/mozmeao.md

```diff
@@ -1,25 +1,67 @@
 # How we work in 2018
 
-Our team manages it's work via a [Github project in the MozMEAO organization](https://github.com/orgs/mozmeao/projects/2). The goal of this team-centric Kanban board is to help manage backend and infra/SRE tasks. This will allow us to:
+#### tl;dr
 
-- prioritize and assign work in a fair and balanced way
+Our team manages it's work via two Github projects separated into [milestones](https://github.com/orgs/mozmeao/projects/3) and [tasks](https://github.com/orgs/mozmeao/projects/2). 
+
+The goal of this process is to:
+
+- establish and follow a team roadmap
+- organize team-wide projects ("swarming") to foster collaboration, prevent isolation, spread knowledge, and bring ⚡️energy⚡️ to cloud engineering
+- plan, prioritize and assign work in a fair and balanced way
 - communicate status within the MozMEAO team and marketing organization
 - identify risks, blockers and high priority issues
-- foster collaboration on the server side
 
 All data in the project is "standalone": we try to keep all links and references as public as possible, but there are obviously tasks that include sensitive data. These sensitive tasks are managed internally, and can also tracked in private Bugzilla issues.
 
 
-# Project board
+# Milestones
+
+Milestones are higher level goals that can contain multiple tasks and can span up to two weeks of work for one or more engineers. Goals that require longer than two weeks of effort should be split up into multiple milestones.
```

Comment:
I'm not sure if the two week time limit makes sense for milestones.

Reply:
Good point. +1 on that. Milestones would normally be > 2 weeks I'd think.

Reply:
The idea here is to use Github milestones as a way to track 2 week chunks of work, rather than thinking of a milestone as a many weeks/months of work.

Reply:
I'm all for time-boxing, but if the boxen have to be exactly "2 weeks" in size then it doesn't seem all that useful and I think we're fine just sticking without milestones.

Reply:
I'm +1 on milestones being longer than 2 weeks, thanks for pointing this out!

I'll update docs shortly.

Reply:
+1 to what y'all said up there.